code/modules/reagents/chemistry/holder.dm

```diff
@@ -299,21 +299,17 @@
 			if(R.addiction_stage < 5)
 				if(prob(5))
 					R.addiction_stage++
-			if(M.reagents.has_reagent(R.id))
```

Comment:
Still need to do this check. But the the negative version. Currently you will display stage 1 addiction stuff even if you have the chem inside of you

Reply:
stage one is the default stage of addiction.

Reply:
Yet is the thing. Somebody can add a new withdrawal type that does something at stage 1.
It's just incorrect in my perspective that you can get withdrawal signs while you're still under influence of the reagent.

Reply:
this is for another PR, this just a straight fix to get smoking and similar methods to sate addiction.

Reply:
You do remove it in this PR. So it is part of the PR. Before it was like this:
Reagent is in body. You don't get symptoms.
Now you can have a reagent in your body and get level 1 symptoms

Reply:
Yeah, this functionality needs to remain.